### DISCUSSION TITLE
Render cliffs

### DISCUSSION THREAD

**Discussion by uhkeller** [organicmaps/organicmaps#445] [Ideas]
It would be nice if OM rendered cliffs ([OSM tag natural=cliff](https://wiki.openstetmap.org/wiki/Tag:natural%3Dcliff)).

**Comment by pwb999** [comment]
It is possible to extract that tag data from OSM and then create a KML file to display the cliff data as your own layer. I did this for the UK Lake District and rendered it in Red. Works ok. Also the UK OS Opendata now has their CRAG data available. However some have noted that the CRAG data is not complete especially in Scotland. It seems ok for the Lakes though.

**Comment by uhkeller** [comment]
Organic maps has been able to render cliffs for a long time now, no need for workarounds. I'd forgotten about this issue, will close it now.
